airframe-sql/src/test/scala/wvlet/airframe/sql/analyzer/TypeResolverTest.scala

```diff
@@ -383,6 +386,30 @@ class TypeResolverTest extends AirSpec {
           fail(s"unexpected plan: ${p}")
       }
     }
+
+    test("resolve having") {
+      val p = analyze("select id, count(*) cnt from A group by id having cnt > 10")
+      p match {
+        case Aggregate(
+              _,
+              _,
+              List(GroupingKey(key1, _)),
+              Some(GreaterThan(col, LongLiteral(10, _), _)),
+              _
+            ) =>
+          key1 shouldBe ra1
+          col shouldBe FunctionCall(
+            "count",
+            Seq(AllColumns(None, Some(Seq(tableA)), Some(NodeLocation(1, 12)))),
+            false,
+            None,
+            None,
+            Some(NodeLocation(1, 12))
```

Comment:
@takezoe It's a bit off-topic from this PR. Do you think if AirSpec supports wildcard match inside case class parameters, will it be useful? https://github.com/wvlet/airframe/issues/2619